docs/conf/atlantic/2023/news/thank-you-recap.rst

```diff
@@ -0,0 +1,122 @@
+.. _Bart at Work: https://www.bartatwork.com/
+.. _YouTube channel: https://www.youtube.com/writethedocs
+
+Sketchnotes
+-----------
+
+We’re excited to share the `full resolution sketchnotes`_ from each talk. You are welcome to use them for non-commercial purposes, like including them in your conference write up ;)
+
+Thanks to `Dennis Dawson`_ for doing them this year!
+
+.. _full resolution sketchnotes: https://flic.kr/ps/
+.. _Dennis Dawson: https://dennissdawson.wixsite.com/mr--dawson/portfolio
+
+Shirts still on sale
+--------------------
+
+If you want a souvenir from this year's conference, our `Write the Docs Atlantic 2023 Pop-Up Shop <https://shirt.writethedocs.org/>`_ is still open until September 30th!
+We have loose and fitted options from different fabrics, and you can choose the size and cut you want.
+
+The campaign will run in 3-day batches, with shipping from the USA or Europe.
+
+Attend Write the Docs Australia 
+-----------------------------------------------
+
+Don’t forget our `Australia conference`_ on **Dec 7-8, 2023**. The CFP is open until September 22nd.
+
+We’re also looking for more sponsors to make our other events just as magical as this year’s Atlantic event. You can see all our `sponsorship options`_ on our website.
+
+.. _Australia conference: https://www.writethedocs.org/conf/australia/2023/
+.. _sponsorship options: https://www.writethedocs.org/sponsorship/
+
+Write the Docs Slack
+--------------------
+
+Our Slack network keeps on growing. Our #general channel in particular has been extra busy in the days and weeks leading up to the conference, and we hope that y’all continue the conversations throughout the year. If you haven’t joined, now is the time! 
+Check out the `Write the Docs Slack`_.
+
+.. _Write the Docs Slack: http://www.writethedocs.org/slack/
+
+Write the Docs Newsletter
+-------------------------
+
+Our `monthly newsletter`_ is the other place to keep track of everything that is happening with the community. 
+Subscribe to make sure you hear about all the conferences, meetups, job postings, and summaries of relevant conversations in our Slack.
+
+.. _monthly newsletter: http://writethedocs.org/newsletter/
+
+Local Meetups
+-------------
+
+We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
+We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+Our `Quorum Meetups`_ are special meetups, which are virtual and target large geographic areas. There’s already two, covering most everyone in the US or in a similar timezone.
+
+.. _build your local or online communities: http://www.writethedocs.org/meetups/
+.. _starting: http://www.writethedocs.org/organizer-guide/meetups/starting/
+.. _Quorum Meetups: https://www.writethedocs.org/organizer-guide/meetups/quorum/
+
```

Comment:
Would it make sense to add a section here about 2024 conferences, or does it deserve its own blog post at some point?

Reply:
Let's make a separate blog post once we have pinned down some more detail.